src/commands/cmd_tdigest.cc

```diff
@@ -168,7 +170,52 @@ class CommandTDigestAdd : public Commander {
   std::vector<double> values_;
 };
 
+enum MinMax { MAX, MIN };
+
+template <MinMax T>
```

Comment:
We can make this simplier and just using argument? This is not a cpu-bounded class, and I guess the performance here is not matter

The compute function uses template because that's cpu bound and branching matters, here we can just take it easy